modules/core/site.js

```diff
@@ -1857,3 +1857,82 @@ function listControlsMenu() {
     $('#list_controls_menu').toggleClass('show')
     $('.list_sources').hide();
 }
+
+
+// Sortablejs
+const tableBody = document.querySelector('.message_table_body');
+let targetFolder;
+let movingElement;
+const sortable = Sortable.create(tableBody, {
+    sort: false,
+    group: 'messages',
+
+    onMove: (sortableEvent) => {
+        movingElement = sortableEvent.dragged;
+        targetFolder = sortableEvent.related?.className.split(' ')[0];
+        return false;
+    }
+});
+
+Sortable.utils.on(tableBody, 'dragend', () => {
+    const page = hm_page_name();
+    const selectedRows = [];
+
+    document.querySelectorAll('.message_table_body > tr').forEach(row => {
+        if (row.querySelector('.checkbox_cell input[type=checkbox]:checked')) {
+            selectedRows.push(row);
+        }
+    });
+    
+    if (selectedRows.length == 0) {
+        selectedRows.push(movingElement);
+    }
+
+    const movingIds = selectedRows.map(row => row.className.split(' ')[0]);
```

Comment:
There is an issue with this line. Try to drap and release the click you will see the issue.
Uncaught TypeError: Cannot read properties of undefined (reading 'className')

Reply:
It seems to be impossible for me to reproduce this error. Any specific indication?